.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Close stale issues and PRs'
+
+on:
+  schedule:
+    # check issue and pull request once every day
+    - cron: '25 11 * * *'
+
+jobs:
+  invalid-stale-close:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          stale-pr-message: 'This PR is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 5 days with no activity.'
+          close-pr-message: 'This PR was closed because it has been stalled for 10 days with no activity.'
+          # only issues/PRS with ALL these labels are checked
+          only-labels: 'invalid, awaiting response'
```

Comment:
Only the issues and pr with both **invalid** and **awaiting response** labels will be checked.
Suppose I to change current strategy to check issues and prs with any of "invalid" and "awaiting response" labels?
According to #8002 prompt message: "This issue is marked as stale because it has been marked as invalid **_or_** awaiting response for 7 days without any further response. It will be closed in 5 days if the stale label is not removed or if there is no further response.", it seems better to do so. @ZwwWayne

Reply:
Yes, sometimes the issue will not be labeled by both of these two labels. Therefore, 'or' might be more appripriate

Reply:
have accomplished the change @ZwwWayne